.gitignore

```diff
@@ -3,6 +3,7 @@
 .bundle
 .config
 .ruby-version
+.tool-versions
```

Comment:
This file is used by [asdf](https://asdf-vm.com/manage/versions.html#set-current-version) which is what I am using for version management. It shouldn't be checked into this repo. I accidentally included it in a previous PR.